scripts/travis/func.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+function travis_time_start {
+    set +x
+    TRAVIS_START_TIME=$(date +%s%N)
+    TRAVIS_TIME_ID=$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 8 | head -n 1)
+    TRAVIS_FOLD_NAME=$1
+    echo -e "\e[0Ktraivs_fold:start:$TRAVIS_FOLD_NAME"
```

Comment:
"traivs" looks like a typo but is consistent through the whole function -- pasted typo or intentional?